lib/components/SButton.vue

```diff
@@ -608,15 +608,16 @@ function handleClick(): void {
 
 .icon-svg {
   fill: currentColor;
-  transition: fill 0.25s;
+  transition-property: color, fill;
```

Comment:
Earlier there were no transitions (color was being modified but fill was set to currentcolor, browsers don't animate in that case unless transition property is color). These changes will animate them, but I don't think we need these transitions. Open SButton >  Icon on Histoire (current deploy) and try changing the color mode (cmd+shift+d). Do the same for preview deploy.